src/pretix/presale/templates/pretixpresale/fragment_calendar_nav.html

```diff
@@ -0,0 +1,28 @@
+{% load i18n %}
+{% load eventurl %}
+{% load urlreplace %}
+
+<div class="btn-group" role="group">
+    <a href="?{% url_replace request "style" "list" "date" "" %}" type="button" 
+            class="btn btn-default{% if style == "list" %} active{% endif %}">
+        {% trans "List" %}
+    </a>
+    <a href="?{% url_replace request "style" "week" "old" "" "page" "" "date" date|date:"Y-\WW" %}" type="button"
```

Comment:
I think you need to use `o` instead of `Y` here to  handle the year boundary correctly. For example, ``1977-01-02`` is part of the week ``1976-W53``

Reply:
Good catch! Thank you!